docs/guides/mssqlserver/backup/application-level/index.md

```
@@ -22,7 +22,7 @@ This guide will give you an overview how you can take application-level backup a
 
 - At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using `Minikube` or `Kind`.
 - Install `KubeDB` in your cluster following the steps [here](/docs/setup/README.md).
-- Install `KubeStash` in your cluster following the steps [here](https://kubestash.com/docs/latest/setup/install/kubestash).
+- Install `KubeStash` in your cluster following the steps [here](https://kubestash.com/docs/latest/setup).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)